src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -342,6 +345,22 @@ public bool MicrobePreviewMode
         }
     }
 
+    /// <summary>
+    ///   Controls the visibility of spatial nodes in the editor.
```

Comment:
Camera is a spatial node and this doesn't hide that, so, this comment is not technically correct. Should probably reword to be about the cell editor, placed cell visuals and editor arrow etc. being hidden

Reply:
I think `graphical spatial nodes` could be more fitting.

Reply:
I'd prefer to mention the cell editor visuals (cell parts) explicitly as I think that gives a better picture as to what this is used for.